language-support/ts/typedoc.bzl

```diff
@@ -15,6 +16,7 @@ def ts_docs(pkg_name):
           # NOTE: we need the --ignoreCompilerErrors flag because we get errors when tsc is trying to
           # resolve the imported packages.
           $(location @language_support_ts_deps//typedoc/bin:typedoc) --out docs --ignoreCompilerErrors --readme README.md --stripInternal $(SRCS)
+          sed -i -e 's/0.0.0-SDKVERSION/{sdk_version}/' docs/**/*.html
```

Comment:
Can we set this to `__VERSION__` instead of `0.0.0-SDKVERSION`? That’s what we use everywhere else so keeping things consistent seems nice.

Reply:
we use `0.0.0-SDKVERSION` everywhere else in the npm package, it needs to be a valid version string.

Reply:
Ah I see, makes sense :+1: